crates/shared/src/gas_price_estimation/mod.rs

```diff
@@ -62,6 +65,21 @@ pub async fn create_priority_estimator(
     for estimator_type in estimator_types {
         tracing::info!("estimator {estimator_type:?}, networkid {network_id}");
         match estimator_type {
+            GasEstimatorType::Driver => {
+                ensure!(
+                    driver_url.is_some(),
+                    "Driver URL must be provided when using Driver gas estimator"
+                );
+                let url = driver_url
+                    .as_ref()
+                    .unwrap()
+                    .parse()
+                    .context("invalid driver URL")?;
```

Comment:
nit, to save some space.
```suggestion
                let url = driver_url
                    .as_ref()
                    .ok_or_else(|| anyhow!("Driver URL must be provided when using Driver gas estimator"))?
                    .parse()
                    .context("invalid driver URL")?;
```